File: PHP/uberPool.php
<?php
require_once('car.php');
class UberPool extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver); // Clase super 
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar()
    {
        echo "Modelo: $this->model Brand: ".$this->brand;
        echo "<br>"; // Salto de linea 
    }

}
?>

File: Java/UberPool.java
package Java;

public class UberPool extends Car { // herencia
    String brand; 
    String model;

    public UberPool(String license, Account driver, String brand, String model){ // metodo constructor
        super(license, driver); // Superclase, hara referencia a los atributos y a los metodos de la superclase
        this.brand = brand; // this hara referencia a los atributos de la clase hija o la clase donde estamos ubicados 
        this.model = model;

        
    }
    
}
